cmd/vclusterctl/cmd/create.go

```diff
@@ -174,11 +174,11 @@ func (cmd *CreateCmd) Run(ctx context.Context, args []string) error {
 		return err
 	}
 
-	output, err := exec.Command(helmBinaryPath, "version", "--client", "--template", "{{.Version}}").CombinedOutput()
-	if errHelm := clihelper.CheckHelmVersion(string(output)); errHelm != nil {
-		return errHelm
-	} else if err != nil {
+	output, err := exec.Command(helmBinaryPath, "version", "--client", "--template", "{{.Version}}").Output()
+	if err != nil {
 		return err
+	} else if errHelm := clihelper.CheckHelmVersion(string(output)); errHelm != nil {
```

Comment:
I guess you could skip the else and reuse `err` from above. So: `errHelm :=` -> `err =`